crates/app/signing/src/collector/extractor_of_instances_required_to_sign_transactions.rs

```diff
@@ -41,8 +39,82 @@ impl ExtractorOfInstancesRequiredToSignTransactions {
     }
 }
 
+#[cfg(test)]
+#[derive(Debug, Default)]
+pub(crate) struct ProtoProfile {
```

Comment:
cfg(test) only used for unit tests in this crate, and we impl `GetEntityByAddress + HasFactorSources` for it, allowing minimal changes to all the unit tests.